cmake/scripts/common/AddonHelpers.cmake

```diff
@@ -121,18 +121,16 @@ macro (build_addon target prefix libs)
             foreach(src_file ${USED_SOURCES})
               file(STRINGS ${src_file} BIN_ADDON_SRC_PARTS)
               foreach(loop_var ${BIN_ADDON_SRC_PARTS})
-                string(FIND "${loop_var}" "#include" matchres)
-                if("${matchres}" EQUAL 0)
-                  string(REPLACE " " ";" loop_var "${loop_var}")
-                  list(GET loop_var 1 include_name)
-                  string(REGEX REPLACE "[<>\"]|kodi/" "" include_name "${include_name}")
-                  if(include_name MATCHES ${depend_header})
+                string(REGEX MATCH "^[ \t]*#[ \t]*(include|import)[ \t]*[\"<].*[\">]" loop_var "${loop_var}")
+                if(loop_var AND loop_var MATCHES "[\"<]kodi/${depend_header}[\">]")
```

Comment:
```suggestion
                string(REGEX MATCH "^[ \t]*#[ \t]*(include|import)[ \t]*[<\"](kodi/)?(.+)[\">]" include_name ${loop_var})
                if(include_name AND CMAKE_MATCH_3 MATCHES ^${depend_header})
```

Reply:
With this change is in conflict with our Jenkins, no idea why but, it seems to give several times not the right "MATCH" and some depends are not added and mainly `ADDON_GLOBAL_VERSION_MAIN_USED` is missing which is mandatory.